### PR TITLE
Don't break if quota command output is empty and fix "uninstall dispatchers" script.

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

**Comment by mpereira** [comment]
This should be good for now but we should probably turn this into a python script in the future so we can take advantage of existing tooling.

File: scale-tests/uninstall-dispatchers.sh [hunk 1]
```diff
@@ -6,39 +6,73 @@ set -x
...
 
 if [ "$#" -lt 1 ]; then
-    echo "Usage: ./uninstall-dispatchers.sh [dispatchers_file]"
+    echo "Usage: ./uninstall-dispatchers.sh [dispatchers_file] [spark_principal]"
     exit 1
 fi
 
-ACS_TOKEN=`dcos config show core.dcos_acs_token`
-PACKAGE_NAME=spark
+readonly DISPATCHERS_FILE="${1:-}"
```
Comment (elezar) [inline scale-tests/uninstall-dispatchers.sh]: Since `DISPATCHERS_FILE` is required, should this not just be `${1}`?

Comment (mpereira) [reply scale-tests/uninstall-dispatchers.sh]: https://github.com/mesosphere/spark-build/pull/306/commits/a141cb3ab284de217a7e1f58b16011d3ef85d85a

File: scale-tests/deploy-dispatchers.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def create_quota(
         existing_quotas = json.loads(stdout)
 
     # remove existing quotas matching name
-    if name in [x['role'] for x in existing_quotas['infos']]:
+    if 'infos' in existing_quotas and name in [x['role'] for x in existing_quotas['infos']]:
```
[the remark this answers is not in the document]
Comment (mpereira) [reply scale-tests/deploy-dispatchers.py]: https://github.com/mesosphere/spark-build/pull/306/commits/c2a09a5f0977ccf9f07267650fb290372d71db33

[the remark this answers is not in the document]
Comment (mpereira) [reply scale-tests/deploy-dispatchers.py]: https://github.com/mesosphere/spark-build/pull/306/commits/4d269015512816f11de696d577eddc2d9ef738f4

File: scale-tests/uninstall-dispatchers.sh [hunk 3]
```diff
@@ -6,39 +6,73 @@ set -x
 # Usage:
 #    (1) Login to cluster via "dcos cluster" command.
 #    (2) Run this script:
-#        ./uninstall-dispatchers.sh [dispatchers_file]
+#        ./uninstall-dispatchers.sh [dispatchers_file] [spark_principal]
 #
 # This script:
 # - uninstalls instances of spark listed in an input file.
 
 if [ "$#" -lt 1 ]; then
```
[the remark this answers is not in the document]
Comment (mpereira) [reply scale-tests/uninstall-dispatchers.sh]: https://github.com/mesosphere/spark-build/pull/306/commits/4d269015512816f11de696d577eddc2d9ef738f4

**Comment by mpereira** [comment]
> @mpereira does this depend on #303?
@elezar it does not, but it will conflict with it.

**Pull request merged**